docs/source/index.rst

```diff
@@ -1,28 +1,340 @@
+                    </div>
+                </div>
+            </div>
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        Introducing the MLflow AI Gateway
+                    </div>
+                    <img class="card-image" src="_static/images/intro/new_features/mlflow-logo.svg" alt="Mlflow"></img>
+                    <div class="body">
+                        We're excited to announce the newest top-level component in the MLflow ecosystem: <strong>The AI Gateway</strong>. 
+                    </div>
+                    <div class="body">
+                        With this new feature, you can create a single access point to many of the most popular LLM SaaS services available now, 
+                        simplifying interfaces, managing credentials, and providing a unified standard set of APIs to reduce the complexity of 
+                        building products and services around LLMs. 
+                    </div>
+                    <div class="doc"><a href="gateway/index.html">Learn more</a></div>
+                    <div class="tag">
+                        <a href="https://github.com/mlflow/mlflow/releases/tag/v2.5.0">released in 2.5.0</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        MLflow Evaluate now supports LLMs
+                    </div>
+                    <img class="card-image" src="_static/images/intro/new_features/mlflow-logo.svg" alt="MLflow"></img>
+                    <div class="body">
+                        You can now use MLflow evaluate to compare results from your favorite LLMs on a fixed prompt.
+                    </div>
+                    <div class="body">
+                        With support for many of the standard evaluation metrics for LLMs built in directly to the API, the featured 
+                        LLM modeling tasks of text summarization, text classification, question answering, and text generation allows you 
+                        to view the results of submitted text to multiple models in a single UI element. 
+                    </div>
+                    <div class="doc"><a href="models.html#model-evaluation-llms">Learn more</a></div>
+                    <div class="tag">
+                        <a href="https://github.com/mlflow/mlflow/releases/tag/v2.4.0">released in 2.4.0</a>
+                    </div>
+                </div>
+            </div>
+        </article>
+    </section>
 
-Get started using the :ref:`quickstart` or by reading about the :ref:`key concepts<concepts>`.
 
 .. toctree::
     :maxdepth: 1
+    :hidden:
 
-    what-is-mlflow
+    introduction/index
+    tutorials/index
+    guides/index
```

Comment:
@BenWilson2 Love what we're doing in this PR. The CSS & embedded notebooks are awesome!

One thing I'd like to adjust / align on is the TOC tree. A couple things I've noticed:

1. We seem to have Quickstarts (https://mlflow.org/docs/latest/quickstart.html), Tutorials, and Guides. The distinction between each is not clear.

2. All of the tutorials are colocated in the same section, which will make it harder to organize things.

**Proposal**: Rather than having Quickstarts, Tutorials, and Guides at the top-level, I propose we structure the TOC tree into topics. Each topic has an overview, guide, and tutorial.

- MLflow Overview

- Getting Started
   - Quickstarts

- LLMs
   - Guide
   - Tutorials
  
- Classical ML
   - Guide
   - Tutorials

- Deep Learning
   - Guide
   - Tutorials

- Production Model Management

- Deployment
   - Guide
   - Tutorials

...

WDYT?

Reply:
Looks good to me :)